bookmarks.html

```diff
@@ -14,99 +14,158 @@
     <title>Quizzzle</title>
   </head>
   <body>
-    <header class="zz-top">
+    <header class="zz-top title">
       <h1>Bookmarks</h1>
     </header>
 
     <main>
       <ul class="card-container" role="list">
-        <li class="card">
-          <div class="question-container">
-            <h2>Question</h2>
-            <p class="question">
+        <li class="card q-card">
+          <div class="q-card__q-container">
+            <h2 class="q-card__title text-primary">Question</h2>
+            <p class="q-card__q-text">
```

Comment:
I wrote things about these class names in another code review. Please have a look.